### ISSUE TITLE
Chunks in 0.14.0 unloaded client-side

### ISSUE THREAD

**Issue by Creeperface01** [PocketMine/PocketMine-MP#3931]
chunks loading is broken in 0.14 and since build 5 chunks dont load

**Comment by Jorti1** [comment]
Are you using a custom phar?

**Comment by Creeperface01** [comment]
Yes

**Comment by Creeperface01** [comment]
But this happens only in 0.14

**Comment by darkspive** [comment]
I agree.

**Comment by darkspive** [comment]
![screenshot_2016-02-15-19-29-28](https://cloud.githubusercontent.com/assets/14966273/13053352/e1550268-d425-11e5-925a-f35b635b0176.png)

**Comment by darkspive** [comment]
It is pc map!
Please add support of pc maps in 0.14

**Comment by beetree** [comment]
I'm having this issue consistently on a fresh clone from the repo.

Doing this:

```
git clone --recursive https://github.com/PocketMine/PocketMine-MP
git submodule update --init
mv PocketMine-MP/src/ src
./start.sh
```

Default `pocketmine.yml` and default `server.properties`.

To recreate, you need to `fly straight north` then `fly straight south` going back. Then the original chunks will not load.

This happens every time.

/b3

**Comment by CreeperLewisCF** [comment]
Yes.

This is a frequent issue occurring. Will be trying to fix it, if I do, I will create a pull request (accept if I do create one). 

Regards,
Chris ~ PHP & C# Dev @ CreeperLewis

**Comment by Incognitov** [comment]
I am also experiencing this issue and it's very common.

**Comment by robske110** [comment]
confirmed x10
Client-sided
It's in all PM Builds, even third-parties
and 0.14 only

**Comment by 64FF00** [comment]
Maybe a client-sided issue?

**Comment by onebone** [comment]
@64FF00 It does not occur in vanilla MCPE server, I think it is server side problem.

**Comment by PEMapModder** [comment]
@robske110 as long as it is fixable, it is not a client-side issue. It is PocketMine's responsibility to follow client changes, not the client's.

**Comment by CreeperLewisCF** [comment]
Agree with @PEMapModder

**Comment by EmeraldGem** [comment]
I haven't experienced any issues with this on any of the NarwhalCraft BETA servers but I have noticed this on others.

**Comment by beetree** [comment]
What server version is narwhalcraft running?

**Comment by 64FF00** [comment]
I'm trying to fix this issue now, will make a new PR when it's finished

**Comment by beetree** [comment]
64, awesome!! :)

**Comment by xxFlare** [comment]
@64FF00 PureChunks :+1:

**Comment by CreeperLewisCF** [comment]
@64FF00 How are you fixing this? 

@beetree @Incognitov @64FF00 I don't think this occurs with Genesys build of PocketMine. You could try that. Maybe LEET could use that?

**Comment by CreeperLewisCF** [comment]
Wondering to try and reproduce in MiNET to see whether client-sided or not.

**Comment by PEMapModder** [comment]
@CreeperLewisCF please do not comment unless what you say helps solving the issue. If you cannot point out the exact difference between PocketMine and genisys that brought the fix, it is meaningless to mention about it.

**Comment by Pub4Game** [comment]
@PEMapModder Check https://github.com/iTXTech/Genisys/commit/75e8cdf5c05fd23b5219d4bc061fb22a53eae494

**Comment by CreeperLewisCF** [comment]
@PEMapModder Respectively, @Beetree has done the same (comments), but I was stopped. This seems more like starting a fight than asking to obey rules (which if I didn't know better; I would have said things back). Please state your statement to all who have done something you oppose of and not just shoot it at one online guest who respects you. 

In addition, you can check the commits on genisys starting from when it updated to PHP7 and so on. It isn't that hard.

**Comment by xxFlare** [comment]
@PEMapModder Just checked with MiNET. Yes, MCPE 0.14 adds Chunk Radius (New packet).

Packet: 0xc9

It's sent by the server, not the client.

And the fields are: Name, Type, Size, Chunk radius, & int.

**Comment by CreeperLewisCF** [comment]
@xxFlare Nicely done. I was going to, but having teency weency problems with a host. 

Also, does PocketMine have this new packet?

**Comment by xxFlare** [comment]
@CreeperLewisCF Well, it's not implemented but @Intyre did note it:

https://github.com/PocketMine/PocketMine-MP/blob/a6cd526e608ed78a5b7e82165a4d790bf0ea852e/src/pocketmine/network/protocol/Info.php#L93

**Comment by Intyre** [comment]
fixed: e11b76318ce2556
